cli/tsc/dts/lib.dom.d.ts

```diff
@@ -22483,6 +22483,8 @@ interface URL {
 declare var URL: {
     prototype: URL;
     new(url: string | URL, base?: string | URL): URL;
+    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/URL/canParse_static) */
+    canParse(url: string | URL, base?: string | URL): boolean;
```

Comment:
We should add tests for both these changes because otherwise this will easily go away in the next TypeScript uprade since these files are generated by TypeScript.

Reply:
Oh, I didn't know. I'll fix in another PR.

Reply:
No problem. Thanks!

Reply:
Could you please point me in the direction of the test that needs to be added?

Reply:
It's ok. It seems these types are in TypeScript now.